storage/client/client.go

```diff
@@ -1303,6 +1303,71 @@ func (c *StorageClient) Validators(ctx context.Context, p apiTypes.GetConsensusV
 		vs.Validators = append(vs.Validators, v)
 	}
 
+	// Validators uptime. TODO: This could be heavily cached or periodically computed in a materialized view.
+	rows, err := c.db.Query(ctx, queries.ValidatorsUptime, address)
```

Comment:
TODO: test the validators uptime query performance on a live mainnet DB, to decide if it needs to be moved into a view.